src/model/roof.mode.ts

```diff
@@ -0,0 +1,8 @@
+export type RoofSurface = {
+  level_1: number
+  level_2: number
+  level_3: number
+  level_4: number
+  favorable: number
```

Comment:
The roof can contain more than 4 level.
Maybe replace level_1,2,3,4 by an array of levels

Reply:
good idea!